website/components/mdx/_download-macOS.mdx

```diff
@@ -24,7 +24,7 @@ chmod a+x cloudquery
 <Tab>
 
 ```bash copy
-curl -L https://versions.cloudquery.io/latest/v2/cloudquery_darwin_arm64 -o cloudquery
+curl -L https://github.com/cloudquery/cloudquery/releases/download/${VERSION_CLI}/cloudquery_darwin_arm64 -o cloudquery
```

Comment:
Why is this link using `${VERSION_CLI}` while the one higher up is using just plain `VERSION_CLI`? Is there a difference?

Reply:
Oh yeah no, looking at the preview site, looks like this is a bug
![Screenshot 2022-11-28 at 09 17 18](https://user-images.githubusercontent.com/1121616/204239613-afac1ae3-5df6-4d35-953e-bb8a98a683f2.png)